categorization-state-machine/README.md

```diff
@@ -0,0 +1,82 @@
+# Message categorizing with Amazon Bedrock and Amazon SQS
+
+This workflow demonstrates how you can use Amazon Bedrock to categorize messages and then send them to an Simple Queue Service queue for further processing. The AWS Serverless Application Model (SAM) template deploys the AWS Step Functions state machine which takes care of the steps related to the workflow. 
```

Comment:
It isn't clear to me reading this that multiple SQS queues are actually used and the right one is chosen by Bedrock